tests/test_validation_passive_context_budget.py

```
@@ -0,0 +1,472 @@
+        f1 = tmp_path / "a.md"
+        f1.write_text("small content", encoding="utf-8")
+        targets = [{"path": "a.md", "max_tokens": 5000, "label": "A"}]
+        results, code = validate_passive_context(tmp_path, targets, ci=True)
+        assert code == 0
+        assert all(r.within_budget for r in results)
+
+    def test_over_budget_ci_returns_one(self, tmp_path: Path) -> None:
+        f1 = tmp_path / "a.md"
+        f1.write_text("x" * 10000, encoding="utf-8")
+        targets = [{"path": "a.md", "max_tokens": 10, "label": "A"}]
+        results, code = validate_passive_context(tmp_path, targets, ci=True)
```

Ruff (0.15.9)

[warning] Unpacked variable `results` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)